src/test/resources/data-sql/create-cart-item.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO cart_items (id, shopping_cart_id, book_id, quantity) VALUES (1, 1, 1, 2);
```

Comment:
![image](https://github.com/user-attachments/assets/3a7c47f7-0914-42f3-82c5-079b18792290)
minor issue but still, always review your files and make sure there's one empty line at the end